packages/nodes-base/nodes/RespondToWebhook/test/RespondToWebhook.test.ts

```diff
@@ -12,10 +15,14 @@ import { RespondToWebhook } from '../RespondToWebhook.node';
 describe('RespondToWebhook Node', () => {
 	let respondToWebhook: RespondToWebhook;
 	let mockExecuteFunctions: MockProxy<IExecuteFunctions>;
+	let mockHelpers: MockProxy<IExecuteFunctions['helpers']>;
 
 	beforeEach(() => {
 		respondToWebhook = new RespondToWebhook();
-		mockExecuteFunctions = mock<IExecuteFunctions>();
+		mockHelpers = mock<IExecuteFunctions['helpers']>({ constructExecutionMetaData });
+		mockExecuteFunctions = mock<IExecuteFunctions>({
+			helpers: mockHelpers,
+		});
```

Comment:
not sure how it will work with types, but you could also write this as

```suggestion
		mockExecuteFunctions = mockDeep<IExecuteFunctions>({
			helpers: { constructExecutionMetaData },
		});
```